code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -5,30 +5,73 @@
 /******BRUTE******/
 /*Suffix: -bital*/
 
-/datum/reagent/medicine/C2/sanguibital
-	name = "Sanguibital"
-	description = "A unique medicine that heals bruises, scaling with the rate at which one is bleeding out. Dilates blood streams, increasing the amount of blood lost. Overdosing further increases blood loss."
-	color = "#ECEC8D" // rgb: 236	236	141
-	taste_description = "whatever vampires would eat"
+/datum/reagent/medicine/C2/helbital //only REALLY a C2 if you heal the other damages but not being able to outright heal the other guys is close enough to damaging
+	name = "Helbital"
+	description = "Named after the norse goddess Hel, this medicine heals the patient's bruises the closer they are to death. Burns, toxins, and asphyxition will increase healing but these damages must be maintained while the drug is being metabolized or the drug will react negatively."
+	color = "#9400D3"
+	taste_description = "cold and lifeless"
 	overdose_threshold = 35
 	reagent_state = SOLID
+	var/datum/status_effect/necropolis_curse/helbent
```

Comment:
Why exactly is this a variable? Couldn't this just be a boolean?

Reply:
I initially thought you could have multiple sources of status effects; Yes